packages/gestalt/src/ButtonToggle/ColorsButton.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { ComponentProps } from 'react';
+import Box from '../Box';
+import ButtonToggle from '../ButtonToggle';
+import Flex from '../Flex';
+import TapArea from '../TapArea';
+
+const BORDER_OFFSET_PX = 4;
+
+function getBorderColor(isHovered: boolean, isSelected: boolean) {
+  // Selection state takes precedence
+  if (isSelected) {
+    return 'selected';
+  }
+  if (isHovered) {
+    return 'tertiary';
+  }
+  return undefined;
+}
+
+export type Props = {
+  colors: ReadonlyArray<string>;
+  isSelected: boolean;
+  onClick?: ComponentProps<typeof ButtonToggle>['onClick'];
+};
+
+export default function SkinToneFilterSwatches({ colors, isSelected, onClick }: Props) {
+  const skinToneCircleHeightPx = 48;
+  const skinToneCircleWidthPx = 88;
+  const filtersContainerHeightPx = skinToneCircleHeightPx + BORDER_OFFSET_PX * 2;
+  const filtersContainerWidthPx = skinToneCircleWidthPx + BORDER_OFFSET_PX * 2;
+
+  const [hovered, setHovered] = React.useState(false);
+
+  return (
+    <Flex alignItems="center" height={filtersContainerHeightPx} justifyContent="start">
+      <TapArea
+        mouseCursor="pointer"
+        onMouseEnter={() => setHovered(true)}
+        onMouseLeave={() => setHovered(false)}
+        // @ts-expect-error - TS2322 - Type '(arg1: TouchEvent<HTMLDivElement>) => void' is not assignable to type 'TouchEventHandler<HTMLButtonElement>'.
```

Comment:
I have managed to change TapArea for the Button tag, this can allow for usage of the hover state for this behavior, I'l going to tinker with this a little bit more tomorrow to see if it works.

Reply:
I've tinkered with it for a bit, decided to keep the hover state to avoid causing differences between current production version.